owncloudApp/src/main/java/com/owncloud/android/utils/FileStorageUtils.java

```diff
@@ -237,4 +237,8 @@ public static boolean deleteDir(File dir) {
 
         return dir.delete();
     }
+
+    public static String getDefaultSourcePath() {
```

Comment:
Let's give it a more descriptive name. `getDefaultCameraSourcePath` could be an option. What do you think?